src/hypofuzz/hypofuzz.py

```diff
@@ -199,21 +211,50 @@ def __init__(
         # Track observability data
         self._last_observed = -math.inf
 
+        self.worker_identity = worker_identity(
+            in_directory=Path(inspect.getfile(self._test_fn)).parent
+        )
+
     def startup(self) -> None:
         """Set up initial state and prepare to replay the saved behaviour."""
         # Report that we've started this fuzz target
         self.db.save(b"hypofuzz-test-keys", self.database_key)
-        # Next, restore progress made in previous runs by loading our saved examples.
-        # This is meant to be the minimal set of inputs that exhibits all distinct
-        # behaviours we've observed to date.  Replaying takes longer than restoring
-        # our data structures directly, but copes much better with changed behaviour.
-        self._replay_queue.extend(self.corpus.fetch())
-        self._replay_queue.append((ChoiceTemplate(type="simplest", count=None),))
+        # save the worker identity once at startup
+        self.db.save_worker_identity(self.database_key, self.worker_identity)
+        # restore our saved minimal covering corpus, as well as any failures to
+        # replay.
+        self._replay_queue.update(self.db.fetch_corpus(self.database_key))
+        if not self._replay_queue:
+            # if no worker has ever worked on this test before, save an initial
+            # GENERATE report, so the graph displays a point at (0, 0).
+            # This is a bit of a hack. It might be better to do this in the
+            # dashboard plotting code instead, by adding a virtual point at (0, 0).
+            self.phase = Phase.GENERATE
+            self._save_report(self._report)
+            self.phase = None
```

Comment:
This does seem like a frontend concern, probably can be mostly handled on the dashboard-server side in linearization logic, and in the browser-side plotting code.  Feels like a followup though.